src/v2/api/controllers/PollController.ts

```diff
@@ -119,38 +181,71 @@ export class PollController {
     return this.questionMapper.getQuestionWithRoles(question);
   }
 
-  @Get('/questions/:questionId')
   @ApiOkResponse({
     type: QuestionWithRolesResponse,
   })
   @ApiBadRequestResponse({
-    description: `InvalidEntityIdException:\n
-                  question with such id is not found`,
+    description: `\n
+    InvalidEntityIdException:
+      question with such id is not found`,
+  })
+  @ApiParam({
+    name: 'questionId', 
+    required: true,
+    description: 'Id of question you want to get',
   })
+  @ApiEndpoint({
+    summary: "Request to get a question by ID",
+  })
+  @Get('/questions/:questionId')
   async getQuestion (
     @Param('questionId', QuestionByIdPipe) questionId: string,
   ) {
     const question = await this.pollService.getQuestionById(questionId);
     return this.questionMapper.getQuestionWithRoles(question);
   }
 
-  @Access(PERMISSION.QUESTIONS_ROLES_GIVE)
   @ApiBearerAuth()
-  @Post('/questions/:questionId/roles')
   @ApiOkResponse({
     type: QuestionWithRolesResponse,
   })
   @ApiBadRequestResponse({
-    description: `InvalidBodyException: Role can not be empty
-                  InvalidBodyException: Visibility parameter is not a boolean
-                  InvalidBodyException: Visibility parameter can not be empty
-                  InvalidBodyException: Requirement parameter is not a boolean
-                  InvalidEntityIdException: question with such id is not found`,
+    description: `\n
+    InvalidBodyException: 
+      Role can not be empty
+
+    InvalidBodyException: 
+      Visibility must be boolean
+
+    InvalidBodyException:
+      Visibility can not be empty
+
+    InvalidBodyException:
+      Requirement must be boolean
+
+    InvalidEntityIdException:
+      Question with such id is not found`,
```

Comment:
* Don't split `InvalidBodyException` into separate parts. There should be only one `InvalidBodyException`  and under it you should write name of your exception, e.g `Visibility cannot be empty`
* Rename: `Role cannot be empty` `Visibility cannot be empty`